pkg/data/accounts.go

```diff
@@ -0,0 +1,104 @@
+package data
+
+import (
+	"fmt"
+
+	"github.com/Optum/dce/pkg/errors"
+	"github.com/Optum/dce/pkg/model"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go/service/dynamodb/expression"
+)
+
+// queryAccounts for doing a query against dynamodb
+func (a *Account) queryAccounts(q *model.Account, keyName string, index string) (*model.Accounts, error) {
```

Comment:
Might move this to a helper function as we add leases and usage.  There isn't much here that is particular to accounts, leases, usage, etc.